gap/PolygonalComplexes/modification.gi

```diff
@@ -1127,6 +1127,164 @@ RedispatchOnCondition( JoinEdgesNC, true,
     [IsTwistedPolygonalComplex,IsList,IsPosInt],
     [IsPolygonalComplex], 0 );
 
+## Faces
+InstallOtherMethod( JoinFaces, "for a polygonal complex and two faces",
+    [IsPolygonalComplex, IsPosInt, IsPosInt],
+    function(complex, F1, F2)
+        return JoinFaces(complex, F1, F2, Faces(complex)[NumberOfFaces(complex)]+1);
+    end
+);
+RedispatchOnCondition( JoinFaces, true, 
+    [IsTwistedPolygonalComplex,IsPosInt,IsPosInt],
+    [IsPolygonalComplex], 0 );
+	
+InstallOtherMethod( JoinFacesNC, "for a polygonal complex and two faces",
+    [IsPolygonalComplex, IsPosInt, IsPosInt],
+    function(complex, F1, F2)
+        return JoinFacesNC(complex, F1, F2, Faces(complex)[NumberOfFaces(complex)]+1);
+    end
+);
+RedispatchOnCondition( JoinFacesNC, true, 
+    [IsTwistedPolygonalComplex,IsPosInt,IsPosInt],
+    [IsPolygonalComplex], 0 );
+
```

Comment:
Some comments would be nice

Reply:
Comments on the implementation or on the various methods?